### PR TITLE
Fix metrics.

### PR DESCRIPTION
**Pull request by fisx**
@tiago writes:

> Start cannon and curl localhost:8083/i/metrics
> 
> Notice there is no net_websocket_clients on the output?
> 
> Now try curl localhost:8083/i/monitoring and again curl localhost:8083/i/metrics.
> 
> Look at what happens! :) Anyway... just wanted to share this... so, culprit
> 
> https://github.com/wireapp/wire-server/blob/develop/services/cannon/src/Cannon/API.hs#L72-L77
> https://github.com/wireapp/wire-server/blob/develop/services/galley/src/Galley/API.hs#L1007-L1010
> 
> Can we please move those to a different place? Calling /i/monitoring should only render metrics and not do some extra logic.

### CODE REVIEW

File: services/cannon/src/Cannon/API.hs [hunk 1]
```diff
@@ -70,11 +69,8 @@ sitemap = do
     head "/i/status" (continue (const $ return empty)) true
 
 monitoring :: Media "application" "json" -> Cannon Response
-monitoring = const $ do
-    m <- monitor
-    s <- D.size =<< clients
-    gaugeSet (fromIntegral s) (path "net.websocket.clients") m
-    json <$> Metrics.render m
+monitoring _ = do
```
Comment (tiago-loureiro) [inline services/cannon/src/Cannon/API.hs]: The `do` can be removed? :P

**Comment by fisx** [comment]
I also just noticed that stern has `/i/monitoring`, but not '/i/metrics`.  Should I fix that, too?  (I'd like to do it in a separate PR, though.)

**Comment by jschaul** [comment]
> I also just noticed that stern has `/i/monitoring`, but not '/i/metrics`. Should I fix that, too? (I'd like to do it in a separate PR, though.)

Yes, please :) (separate PR sounds good)

File: services/cannon/src/Cannon/API.hs (hunk 1 above)
Comment (fisx) [reply services/cannon/src/Cannon/API.hs]: i was tempted to argue against that, but fine...  :)

**Comment by fisx** [comment]
One more problem: `waiPrometheusMiddleware` creates its own monitoring handle, which doesn't get updated by `refreshMetrics`.  One more commit coming up!

**Comment by fisx** [comment]
never mind my last commit.  we're using the same state in prometheus-client and our own metrics-core, metrics-wai.  i currently don't fully understand why, but it's anyway unrelated to this PR...

**Comment by fisx** [comment]
ci failure: flaky gundeck test again, unrelated.

**Comment by fisx** [comment]
again...

**Pull request merged**